index.d.ts

```diff
@@ -0,0 +1,114 @@
+ */
+export function identify(identity: string): void;
+
+/**
+ * Attach custom properties to user profiles.
+ *
+ * The `addUserProperties` API lets you attach custom properties to user profiles,
+ * such as account-level info from your database, or demographic info.
+ *
+ * To better understand the client-side `identify` and `addUserProperties` APIs,
+ * [take a look at our comprehensive guide.](https://docs.heapanalytics.com/docs/using-identify)
+ *
+ * @param properties a JSON object containing key-value pairs to be associated with a user.
+ * Keys and values must be a number or string, with the value being 255 characters or fewer.
+ *
+ * Also, the string user_id cannot be used as a key in the user properties object.
+ * For Android, the property map keys and values must be converted to strings,
+ * but they can be queried as numbers in Heap.
+ *
+ * User properties are associated with all of the user's past activity, in addition to their future activity.
+ * Custom user properties can be queried in the same fashion as any other user property.
+ *
+ * __Important:__ If you want to write your user's email into the builtin Email property,
+ * you must send the key lowercase i.e. heap.addUserProperties({"email": `"user@example.com"`});
+ * Sending an email value with a non-lowercase key will create a new property to store this data.
+ */
+export function addUserProperties(properties: object): void;
+
+/**
+ * Specify a set of global key-value pairs to get attached to all of a user's subsequent events.
+ *
+ * These event properties will persist across multiple sessions on the same device and get applied to both
+ * auto-captured and custom events.
+ *
+ * This is useful if you have some persistent state, but you don't want to apply it across all of a user's
+ * events with `identify`. A good example is "Logged In", which changes over the user's lifecycle.
+ * You can use `addEventProperties` to measure how a user's behavior changes when they're logged in
+ * vs. when they're logged out.
+ *
+ * @param properties a JSON object containing key-value pairs to be associated with
+ * every subsequent event. Keys and values must be a number or string fewer than 1024 characters.
+ *
+ * For Android, the property map keys and values must be converted to strings,
+ * but they can be queried as numbers in Heap.
+ */
+export function addEventProperties(properties: object): void;
+
+/**
+ * Stops a single event property from getting attached to all subsequent events.
+ *
+ * @param property name of the event property to remove. This stops attaching the property to all subsequent events.
+ */
+export function removeEventProperty(property: string): void;
+
+/**
+ * Removes all stored event properties.
+ */
+export function clearEventProperties(): void;
+
+/**
+ * Changes the frequency with which events are sent back to the Heap server from an iOS mobile device.
```

Comment:
I think this is a copy-paste mistake. I don't think we expose `changeInterval` yet.

Reply:
The old React Native library had exposed it on iOS, but Android only had it as a private function (IIRC).  To keep the same interface, we dropped that from the initial version.

Reply:
My mistake, just removed it.